src/task/MonitorTask.ts

```diff
@@ -306,18 +307,41 @@ export default class MonitorTask
       }
     );
 
-    this.applyHealthPolicy(
-      this.sendingAudioFailureHealthPolicy,
-      connectionHealthData,
-      () => {
-        const attributes = this.generateBaseAudioVideoEventAttributes();
-        this.context.eventController?.publishEvent('sendingAudioFailed', attributes);
-      },
-      () => {
-        const attributes = this.generateBaseAudioVideoEventAttributes();
-        this.context.eventController?.publishEvent('sendingAudioRecovered', attributes);
-      }
-    );
+    if (this.shouldEvaluateSendingAudioFailurePolicy) {
+      this.applyHealthPolicy(
+        this.sendingAudioFailureHealthPolicy,
+        connectionHealthData,
+        () => {
+          const attributes = this.generateBaseAudioVideoEventAttributes();
+          this.context.eventController?.publishEvent('sendingAudioFailed', attributes);
+        },
+        () => {
+          const attributes = this.generateBaseAudioVideoEventAttributes();
+          this.context.eventController?.publishEvent('sendingAudioRecovered', attributes);
+        }
+      );
+    }
+  }
+
+  audioVideoDidStart(): void {
+    // We want to wait until we're connected to avoid false-positives.
+    this.shouldEvaluateSendingAudioFailurePolicy = true;
+  }
+
+  audioVideoDidStartConnecting(reconnecting: boolean): void {
+    // Similar to the start of a meeting, we don't want to evaluate sendingAudioFailureHealthyPolicy
+    // until we're connected. The expectation here is that the flag will be set to true again when
+    // audioVideoDidStart() is eventually called.
+    /* istanbul ignore else */
+    if (reconnecting) {
```

Comment:
nit: Couldn't we always set this to false ? No need for the if check.

Reply:
Agreed, I just wanted to make it more explicit in the context of the flag's purpose, but as you suggested below I'm changing the variable name to `isMeetingConnected` so I can remove the if condition here and just set it to false.